import_export/formats/base_formats.py

```diff
@@ -184,3 +201,29 @@ def create_dataset(self, in_stream):
         for i in moves.range(1, sheet.nrows):
             dataset.append(sheet.row_values(i))
         return dataset
+
+
+class XLSX(TablibFormat):
+    TABLIB_MODULE = 'tablib.formats._xlsx'
+    CONTENT_TYPE = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+
+    def can_import(self):
+        return XLSX_IMPORT
+
+    def create_dataset(self, in_stream):
+        """
+        Create dataset from first sheet.
+        """
+        assert XLSX_IMPORT
+        from io import BytesIO
+        from openpyxl import load_workbook
```

Comment:
Oh, I see that tablib bundled openpyxl does not include load_workbook:
https://github.com/kennethreitz/tablib/issues/167